octobot_tentacles_manager/util/tentacle_explorer.py

```diff
@@ -36,7 +37,11 @@ def get_tentacles_from_package(tentacles, package_name: str):
 
 def _load_all_metadata(tentacles):
     for tentacle in tentacles:
-        tentacle.sync_initialize()
+        try:
+            tentacle.sync_initialize()
+        except Exception as e:
+            logging.get_logger(__name__).error(f"Error when loading {tentacle} ({tentacle.tentacle_path}) metadata: {e}")
+            raise e
```

Comment:
Should we keep this `raise`?

Reply:
Yes, as previously, we should not hide exceptions (here we just log an error prior to raising)